### ISSUE TITLE
Canned responses , add to bottom

### ISSUE THREAD

**Issue by siomosp** [mike-koch/Mods-for-HESK#464]
Hello,
Since the last update the Canned responses always replace message ,although add to bottom is selected

**Comment by mike-koch** [comment]
Yes, there is an issue with the message being replaced.  A temporary workaround is to enable "[Rich Text for Tickets](https://modsforhesk.atlassian.net/wiki/display/MFH/Rich+Text+Editor+for+Tickets)", as there are no issues with canned responses there.

If you choose to enable Rich Text for Tickets, please note that you may need to reformat and save your templates as they need to be properly formatted.

**Comment by siomosp** [comment]
Thanks a lot!
